pkg/controllers/dashboard/clusterregistrationtoken/status.go

```diff
@@ -37,11 +37,12 @@ func (h *handler) isRKE2(clusterID string) bool {
 }
 
 func (h *handler) assignStatus(crt *v32.ClusterRegistrationToken) (v32.ClusterRegistrationTokenStatus, error) {
-	ca := systemtemplate.CAChecksum()
+	checksum := systemtemplate.CAChecksum()
+	ca := ""
```

Comment:
What if the checksum is empty? Does the empty CA get consumed? If so, what implications does that have?

Reply:
it ends up as an empty string at the end of the command, look at %s%s stuff in the sprintf format

Reply:
I see that, but I meant more of the technical implications.  Just want to make sure because it looks like in the way that it's currently written, `ca` and `caWindows` will _always_ be assigned a non-empty value.

Reply:
how? ca and caWindows will only be assigned a value if checksum isn't empty.

Reply:
they were assigned to empty strings before. why would it always be non-empty? cachecksum sometimes `== ""` if we do --no-cacerts on rancher. the only fix here is to not add the suffix to the caWindows of `--ca-checksum` because ca was getting overwritten right above it.

Reply:
Misread the original function that was called for CA assignment. LGTM thanks for the explanation